docs/api-testing/composer.md

```diff
@@ -2,82 +2,601 @@
+  * [Logical Test Components](/api-testing/composer/logical-components/)
+
+#### Add an I/O Request Test Component
+To create a simple `GET` request and validate that response is correct:
+  1. In API Testing, on the **Compose** page, click the **Add component** button.
+
+  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen" width="600"/>
+
+  2. In the list of component options, click the **GET** component.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-nav.png')} alt="Navigating to the GET request window" width="400"/>
+
+  3. In the **GET request** window, in the **Url** field, enter **ht<span>tps://</span>api.us-west-1.saucelabs.com/rest/v1/public/tunnels/info/versions**.
+
+  This URL will return a json response body.
+  4. In the **Variable** field, enter **payload**. This variable stores the response.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-window.png')} alt="Editing in the GET request window"/>
+
+  5. Leave the rest of the fields blank and then click the **Confirm changes** icon.
+
+  <img src={useBaseUrl('/img/api-testing/get-request-confirm-changes.png')} alt="Confirm changes icon" width="200"/>
+
+  The result should look like the following:
+
+  <img src={useBaseUrl('/img/api-testing/get-request-final.png')} alt="What the GET request should look like" width="600"/>
 
-All test runs appear to the right of the Composer, underneath the test details and environment sections.
-<img src={useBaseUrl('img/api-fortress/2021/01/testRuns.png')} alt="Test Runs Section" />
+For more information, see [I/O Request Test Components](/api-testing/composer/io-components/).
+
+#### Add an Assertion Component
+1. In API Testing, on the **Compose** page, click the **Add component** button.
+
+  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen" width="600"/>
+
+2. In the list of component options, click the **Assert Exists** component.
+
+  <img src={useBaseUrl('/img/api-testing/assert-exists-nav.png')} alt="Navigating to the Assert exists window"/>
+
+3. In the **Assert exists** window, in the **Expression** field, enter `payload.downloads`. This expression checks for the **downloads** field in the json response body.
+
+4. Leave the rest of the fields blank and click the checkmark to confirm the changes.
+
+  <img src={useBaseUrl('/img/api-testing/assert-exists-window.png')} alt="Confirm changes"/>
+
+5. The result should look like the following:
+
+  <img src={useBaseUrl('/img/api-testing/assert-exists-final.png')} alt="What the Assert request should look like" width="500"/>
+
+For more information, see [Assertion Test Components](/api-testing/composer/assertion-components/).
+
+### Run the Test
+In the Composer, click the **Save** icon and then click **Run**.
+
+<img src={useBaseUrl('/img/api-testing/run-test-save-run.png')} alt="Save and Run icons in the Composer" width="500"/>
```

Comment:
@LoryApiFortress If you want to have a discussion offline about screenshot standards and guidelines, please set something up. Otherwise, I ask that you trust me to do my job.